app/mailers/invitation_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class InvitationMailer < ActionMailer::Base
+  def new_invitation(invitation)
+    @invitation = invitation
+    @member = invitation.member
+    @organization = @member.organization
+
+    mail(to: @invitation.email, subject: 'Invitation', from: '<no-reply@cashbox.dev>')
```

Comment:
@Gataniel or move i t to config, like here: http://stackoverflow.com/questions/2445100/actionmailer-default-from-address